src/components/core/tests/PivotTable.spec.tsx

```diff
@@ -27,6 +26,7 @@ import { waitFor } from "../../filters/AttributeFilter/tests/utils";
 import { IGridCellEvent } from "../pivotTable/agGridTypes";
 import { IDrillEventExtended } from "../../../interfaces/DrillEvents";
 import { Execution } from "@gooddata/typings";
+import noop = require("lodash/noop");
```

Comment:
Is it correct import? Cannot we use `import {noop} from "lodash";`?

Reply:
We do it this way on other places to avoid full lodash import